### PR TITLE
new ini switch added

### PR DESCRIPTION
**Pull request by uday610**
New ini switch documentation

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: src/runtime_src/doc/toc/xrt_ini.rst [hunk 1]
```diff
@@ -37,41 +37,52 @@ The following table lists all supported groups, keys, valid key values, and shor
...
+| polling_throttle    | [N]                          |Specify the time interval in microseconds  |
+|                     |                              |that the runtime library polls the device. |
+|                     |                              |                                           |
+|                     |                              |Default: 0                                 |
++---------------------+------------------------------+-------------------------------------------+
+| verbosity           | [0|1|2|3]                    |verbosity level of log messages. Higher    |
+|                     |                              |number implies more verbosity              |
+|                     |                              |                                           |
+|                     |                              |Default: 0                                 |
++---------------------+------------------------------+-------------------------------------------+
```
Comment (stsoe) [inline src/runtime_src/doc/toc/xrt_ini.rst:74]: @uday610 Let's fix this one.   verbosity has many more values and default is 4.

File: src/runtime_src/doc/toc/xrt_ini.rst [hunk 2]
```diff
@@ -37,41 +37,52 @@ The following table lists all supported groups, keys, valid key values, and shor
...
+|                     |                              |Default: console                           |
++---------------------+------------------------------+-------------------------------------------+
+| cpu_affinity        | [{N,N,...}]                  |Pin all runtime threads to specified CPUs. |
+|                     |                              |                                           |
+|                     |                              |Example: cpu_affinity = {4,5,6}            |
++---------------------+------------------------------+-------------------------------------------+
+| polling_throttle    | [N]                          |Specify the time interval in microseconds  |
+|                     |                              |that the runtime library polls the device. |
+|                     |                              |                                           |
+|                     |                              |Default: 0                                 |
```
Comment (stsoe) [inline src/runtime_src/doc/toc/xrt_ini.rst:68]: This is only applicable for userspace software scheduler which is for internal use only.  Lets remove from documentation.

File: src/runtime_src/doc/toc/xrt_ini.rst (hunk 1 above)
Comment (uday610) [reply src/runtime_src/doc/toc/xrt_ini.rst]: @stsoe  , done. But I think the way the internal code is written and this switch name is confusing. 
If I see https://github.com/Xilinx/XRT/blob/master/src/runtime_src/core/include/xrt.h#L219 , xclVerbosityLevel is still 4, but this switch does not correspond to that, though from the name it seems like. 
This switch corresponds to xrtLogMsgLevel (https://github.com/Xilinx/XRT/blob/master/src/runtime_src/core/include/xrt.h#L203 ). So the name of the switch better fits "loglevel" or something like this.

File: src/runtime_src/doc/toc/xrt_ini.rst (hunk 2 above)
Comment (uday610) [reply src/runtime_src/doc/toc/xrt_ini.rst]: done. Thanks

File: src/runtime_src/doc/toc/xrt_ini.rst (hunk 1 above)
Comment (stsoe) [reply src/runtime_src/doc/toc/xrt_ini.rst]: yes that's confusing.  i will try look at that and clean it up.  thanks!

**Pull request merged**